conf/accumulo-testing.properties

```diff
@@ -82,6 +82,9 @@ test.ci.ingest.pause.wait.max=180
 test.ci.ingest.pause.duration.min=60
 # Maximum pause duration (in seconds)
 test.ci.ingest.pause.duration.max=120
+# The probability (between 0.0 and 1.0) that a set of entries will be deleted during continuous ingest
+# To disable deletes, set probability to 0
+test.ci.ingest.delete.probability=0.1
```

Comment:
The last thing I'm not so sure on is what the default value of this should be. Should we have deletes disabled by default?

Reply:
> The last thing I'm not so sure on is what the default value of this should be. Should we have deletes disabled by default?

People use continuous ingest for different things.  It was created to test the correctness of Accumulo at scale and this is what I think of as its primary use case.  Its secondary use case is that sometimes people run it just to generate load and/or data.  For the primary use case having deletes on by default seems good because it tests more functionality.  For the secondary use case having deletes on may be less desirable.  So since it seems good for primary use case we should probably have something greater than zero. As for what value >0 it should be, I am not sure.  I like the default you have chosen, seems like a good starting point.  We can change it later if we gain some more insight from running with this new feature.